apps/cookbook/src/app/examples/item-example/examples/simple/multiline.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-item-example-simple-multiline',
+  template: `<kirby-item>
+  <kirby-label>
+    <h3>Title</h3>
+    <p wrap subtitle> With the 'wrap' attribute - text no longer faces the risk of abrupt truncation. With 'wrap' in the mix, the content gracefully extends to the next line, seamlessly continuing its narrative. Take a glance, and you'll see how seamlessly it moves to the next line, maintaining readability and preserving the flow of information.</p>
```

Comment:
LOOOOOL 🤣 👏🏻 

Consider wrapping (🥁) the `wrap` attribute in a code block to make it stand out: 
```suggestion
    <p wrap subtitle> With the 'wrap' attribute - text no longer faces the risk of abrupt truncation. With <code>'wrap'</code> in the mix, the content gracefully extends to the next line, seamlessly continuing its narrative. Take a glance, and you'll see how seamlessly it moves to the next line, maintaining readability and preserving the flow of information.</p>
```